src/Plugin.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+namespace RocketCDN;
+
+use RocketCDN\Dependencies\League\Container\Container;
+use RocketCDN\EventManagement\EventManager;
+
+class Plugin {
+	/**
+	 * Is the plugin loaded
+	 *
+	 * @var boolean
+	 */
+	private $loaded = false;
+
+	/**
+	 * Checks if the plugin is loaded
+	 *
+	 * @return boolean
+	 */
+	private function is_loaded(): bool {
+		return $this->loaded;
+	}
+
+	/**
+	 * Loads the plugin in WordPress
+	 *
+	 * @return void
+	 */
+	public function load() {
+		if ( $this->is_loaded() ) {
+			return;
+		}
+
+		$container = new Container();
```

Comment:
don't we need to add a filter here so we can use on support helper plugins? like what we do at WPR plugin `rocket_container`? what do u think?

Reply:
Updated the code for that 👍🏼